src/components/react-hook-form/SelectItemFormField.stories.tsx

```diff
@@ -16,13 +16,13 @@ import { z } from 'zod'
 import {
   InternationalizationMessages,
   Theme,
-} from '../../../../.storybook/components'
-import { ErrorBody } from '../../util'
-import { Form } from '../Form'
+} from '../../../.storybook/components'
+import { ErrorBody } from '../util'
+import { Form } from './Form'
 import {
   SearchQueryParameters,
   PaginatedResponse,
-} from './SelectItemDialogWithSearch'
+} from './SelectItem/SelectItemFormFieldDialog'
```

Comment:
```suggestion
} from './SelectItemFormField/SelectItemFormFieldDialog'
```

Why did you not use `SelectItemFormField` since this is the common base?

Reply:
Because it makes more sense to me. But if you insist I can change it to `SelectItemFormField`

Reply:
That's why I though of leaving all the related files inside this folder:
- SelectItemFormField.tsx
- SelectItemFormField.stories.tsx
- SelectItemFormfieldDialog.tsx
- SelectItemFormFieldInput.tsx

For a user/developer this would mean that he finds everything inside one single folder called `SelectItemFormField`. This is a component that requires more than one file for us to implement it, but we still try to strick to the naming convention. (We do the same in other places: https://github.com/aboutbits/react-ui/tree/main/src/components/dialog/DialogHeader).

I just suggested to place the story inside storybook on the same level as the other fields, so that a user/developers finds the component inside the same directory as the others.

Reply:
This causes errors when trying to export both the component `SelectItemFormField` and the folder `SelectItemFormField`. With `DialogHeader` it works because the component `DialogHeader` is inside and not adjacent to the folder.